eng/.docsettings.yml

```diff
@@ -25,7 +25,7 @@ language: net
 root_check_enabled: True
 
 required_readme_sections:
-  - ".+ client library for .NET"
+  - "Azure .+ client library for .NET"
```

Comment:
Adding this because I missed it in the [last PR](https://github.com/Azure/azure-sdk-for-net/pull/39335) and it's needed to address [this comment](https://github.com/Azure/azure-sdk-for-net/pull/39135#discussion_r1362639669) - not related to tests.